### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds.com#19]
For example, from # 19: Some background/foreground colour combinations with an insufficient contrast ratio were found in the main content area of the page: 
For Example,
• The #FEFCFE image of text "We" has a contrast ratio of 2.1:1  with #97BF0F background.
• The #FDFFF6 image of text "want" has a contrast ratio of 3.2:1 with #0097DA background.
• The #FCFAF9 image of text "Your"  has a contrast ratio of 1.7:1 with #F9B512 background.
• The #FDFBFE image of text "feedback!" has a contrast ratio of 3.5:1 with #E64E98 background.

Including but not limited to - please check for this existing elsewhere:

Page Name | Accessibility Report Issue Number
------------|-----------------------------------
Communications | # 19

Numbers are the issue numbers from the Accessibility Report on the Findings worksheet at https://docs.google.com/spreadsheets/d/1fQgaUcbWK5n8gUorLlAoITfGaJC0Gqg1jzuur9wPoZg/edit#gid=585536028
